base-images/julia1.8rc1-ubuntu20.04.Dockerfile

```diff
@@ -0,0 +1,125 @@
+ARG ENVD_VERSION
+ARG ENVD_SSH_IMAGE
+FROM ubuntu:20.04 as base
+
+FROM base as base-amd64
+
+FROM base as base-arm64
+
+FROM ${ENVD_SSH_IMAGE}:${ENVD_VERSION} AS envd
+
+FROM base-${TARGETARCH}
+
+LABEL maintainer "envd-maintainers <envd-maintainers@tensorchord.ai>"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV PATH="/usr/bin:${PATH}"
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends --no-install-suggests --fix-missing \
+    bash-static libtinfo5 libncursesw5 apt-utils \
+    # conda dependencies
+    bzip2 ca-certificates libglib2.0-0 libsm6 libxext6 libxrender1 mercurial \
+    procps subversion wget \
+    # envd dependencies
+    python3 curl openssh-client git tini sudo python3-pip zsh vim \
+    && rm -rf /var/lib/apt/lists/*
+
+# Leave these args here to better use the Docker build cache
+ARG CONDA_VERSION=py39_4.11.0
+
+RUN set -x && \
+    UNAME_M="$(uname -m)" && \
+    if [ "${UNAME_M}" = "x86_64" ]; then \
+        MINICONDA_URL="https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh"; \
+        SHA256SUM="4ee9c3aa53329cd7a63b49877c0babb49b19b7e5af29807b793a76bdb1d362b4"; \
+    elif [ "${UNAME_M}" = "s390x" ]; then \
+        MINICONDA_URL="https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}-Linux-s390x.sh"; \
+        SHA256SUM="e5e5e89cdcef9332fe632cd25d318cf71f681eef029a24495c713b18e66a8018"; \
+    elif [ "${UNAME_M}" = "aarch64" ]; then \
```

Comment:
Conda is not needed?

Reply:
Yep I think we can remove it. I was thinking if we should support python in julia's image Ref [#same](https://github.com/tensorchord/envd/issues/407)